docs(API): add import.meta section (#6761)
webpack/webpack.js.org: src/content/api/module-variables.mdx

If used inside an expression that is parsed by the Parser, the configuration option is treated as `true`.

## import.meta

The `import.meta` exposes context-specific metadata to a JavaScript module, such as the URL of the module. It is only available in [ESM](/guides/ecma-script-modules/).

> Note that accessing `import.meta` directly is not supported by webpack, you need to use `import.meta` as a property access or through destructuring assignment. E.g.,

Reviewer: ```suggestion
Note that accessing `import.meta` directly is not supported by webpack, you need to use `import.meta` as a property access or through the destructuring assignment. E.g.,
```

> Note that accessing `import.meta` directly is not supported by webpack, you need to use `import.meta` as a property access or through the destructuring assignment. E.g.,

Author: ```suggestion
Please note that webpack does not support direct access to `import.meta`. Instead, you should access its properties or use destructuring assignment. E.g.,
```
